pilot/pkg/model/telemetry.go

```diff
@@ -404,23 +406,27 @@ func (t *Telemetries) telemetryFilters(proxy *Proxy, class networking.ListenerCl
 // mergeLogs returns the set of providers for the given logging configuration.
 // This currently is just the names of providers as there is no access logging configuration, but
 // in the future it will likely be extended
-func mergeLogs(logs []*tpb.AccessLogging, mesh *meshconfig.MeshConfig) sets.Set {
+func mergeLogs(logs []*tpb.AccessLogging, mesh *meshconfig.MeshConfig) (sets.Set, *tpb.AccessLogging_Filter) {
 	providers := sets.NewSet()
 
 	if len(logs) == 0 {
 		for _, dp := range mesh.GetDefaultProviders().GetAccessLogging() {
 			// Insert the default provider.
 			providers.Insert(dp)
 		}
-		return providers
+		return providers, nil
 	}
-
+	var alsFilter *tpb.AccessLogging_Filter
 	providerNames := mesh.GetDefaultProviders().GetAccessLogging()
 	for _, m := range logs {
 		names := getProviderNames(m.Providers)
 		if len(names) > 0 {
 			providerNames = names
 		}
+
+		if m.Filter != nil {
+			alsFilter = m.Filter
```

Comment:
Is it feasible to validate the `m.Filter` is valid CEL expression? Using cel-go perhaps? To add to the validating webhook, not this code.

Reply:
I will try to find out whether it's feasible to add a validating webhook, maybe another PR.

[Filter](https://github.com/istio/api/blob/7bbe5627adf02619660781bb88b01ee4c8af2989/telemetry/v1alpha1/telemetry.proto#L591) is optional, but I'm not sure the merge rule is suitable when there are multi Telemetry contains filter on target `Proxy`, right now it depends on the `logs` array order.